src/RootFS.jl

```diff
@@ -124,15 +179,33 @@ function update_rootfs(triplets::Vector{S}; automatic::Bool = automatic_apple,
         end
 
         if squashfs
-            # If squashfs, verify/redownload the squashfs image
             squashfs_path = downloads_dir("rootfs-$(shard_name).squashfs")
-            download_verify(
+            
+            
+            file_existed = isfile(squashfs_path)
+            
+            # If squashfs, verify/redownload the squashfs image
+            if !(file_existed && download_verify(
                 url,
                 hash,
                 squashfs_path;
                 verbose = verbose,
-            )
-
+                force = true
+            ))
+                # Unmount the mountpoint. It may point to a previous version
+                # of the file. Also, we're about to mess with it
+                if success(`mountpoint $(dest_dir)`)
```

Comment:
We should probably unmount it before running `download_verify`, right?  Since we may attempt to overwrite the file.

Reply:
Overwriting the file is fine, the kernel keeps a reference to it however, so the mountpoint would reflect the old version.